src/components/characterList.jsx

```diff
@@ -7,12 +7,14 @@ const CharacterList = ({ currentScript }) => {
   // get the character data on the initial load and when the script option changes
   useEffect(() => {
     // fetch request to get the character data
-    fetch('/script/' + currentScript + '/characters')
-      .then((response) => response.json())
-      .then(setCharacters)
-      .catch((error) => {
-        console.error(`error: ${error} when fetching character data`);
-      });
+    if (currentScript !== null) {
```

Comment:
Since initial page load does not have any scripts selected, avoid making a request for character info that is sure to fail.